Source/Windows/Common/CPUFeatures.cpp

```diff
@@ -57,7 +65,13 @@ FEXCore::HostFeatures CPUFeatures::FetchHostFeatures(bool IsWine) {
 
   RegCloseKey(Key);
 
-  return FEX::FetchHostFeatures(Features, !IsWine, CTR, MIDR);
+  auto HostFeatures = FEX::FetchHostFeatures(Features, !IsWine, CTR, MIDR);
+  if (IsWine) {
+    FEX::FillMIDRInformationViaLinux(&HostFeatures);
```

Comment:
You can drop this path, will implement a generic way as a follow up

Reply:
Sure thing, removed that path for you